digital-deck-ui/src/components/OtherPlayers.vue

```diff
@@ -0,0 +1,46 @@
+<template>
```

Comment:
Can you rename this to something like `PlayerList` please? `OtherPlayers` sound weird.